### PR TITLE
496: Factor out E, reuse _ arg

### PR DESCRIPTION
**Pull request by subzey**
_I hope I don't bother you too much with my PRs. I like this project so much I just can't get it out of my head._

Factored out `-i%2`. This change adds one more statement, `E=-i%2`, but overall it saves 1 byte.

An argument of `_` is reused as `s`. Other statements are rearranged so `_` is now always called with `0`:
- 3rd param of `setTimeout` (a 1st param passed to the callback) is now `M`v-E`&&M`v`&(p=k=0)` which is always evaluated to `0`.
- Initial call with `S=P=k=0`

### CODE REVIEW

**Comment by veu** [comment]
You don’t bother me at all. I hope I get enough room to implement the bag of seven.

File: src/tetris.js [hunk 1]
```diff
@@ -15,30 +15,30 @@
...
+(_=s=>D(
     P=P||[...'021213030405061617'.substr(
         new Date%6*2,
         8,
-        s=0,
         B=[...C,...b].filter(v=>v^1023||!(S+=++s)).slice(-18)
     )],
-    M`v-~i%2`&&M`v`&(p=k=0),
-    setTimeout(_,k-2&&200)
+    setTimeout(_,k-2&&200,M`v-E`&&M`v`&(p=k=0))
```
Comment (veu) [inline src/tetris.js]: Unfortunately, you can’t combine the statements like that. `k` has to be set before `k-2` is evaluated or the pause to show removed lines will not happen for accelerated pieces.

**Comment by aemkei** [comment]
@subzey: have you seen https://justecorruptio.github.io/golf/tetris.html ?

**Comment by veu** [comment]
I’ll merge your PR if you just pass `0` as third argument to `setTimeout`. That should put it at 498b which is still great.

**Comment by aemkei** [comment]
Also, if you use `<pre>` it will be ~16 bytes shorter:
```js
<pre id=Z><script> 
... 
Z.innerText=S+=v^1023+P&&v>>x&1?"X ":". "
...
</script>

**Comment by veu** [comment]
I don’t want to just copy features from the other tetris. Especially when it changes the way it looks completely.

**Comment by subzey** [comment]
@veu Here's a fixed commit. (Force-pushed over the previous one for the cleaner git history)

**Comment by subzey** [comment]
I'm sure something can be done later with that trailing 0. Or the `(S+=++s)` might change in the future

**Pull request merged**